src/api/app/views/webui2/webui/monitor/_workers_table.html.haml

```diff
@@ -1 +1,49 @@
-%h2 Workers
+#workers-form
```

Comment:
I didn't see this form's dropdown at first (it's quite far on the right on a desktop).

I would move this form below the `h2` and add a bit of margin.

Now:
![workers-now](https://user-images.githubusercontent.com/1102934/58870921-62729f80-86c1-11e9-9d8a-e549c4890f39.png)

My suggestion:
![workers-suggestion](https://user-images.githubusercontent.com/1102934/58870896-5090fc80-86c1-11e9-99c0-3e0466872824.png)

I removed the `display: inline` on the form, removed the `float: right` on the div and applied some margin on the div with the class `my-2`.

Reply:
I don't want redesign this page now. So I would let it to after the migration.